src/app/about/page.tsx

```diff
@@ -0,0 +1,502 @@
+                                        membership either online on our website, or in person at a
+                                        club event or the duck lounge.
+                                    </p>
+                                }
+                                colour={'purple'}
+                            ></FAQ>
+                            <FAQ
+                                question={'Where and what is the Duck Lounge?'}
+                                answer={
+                                    <p>
+                                        The Duck Lounge is located at EM110 in the Engineering and
+                                        Mathematics Building. It has a study space with power
+                                        outlets and couches, along with a Nintendo Switch for games.
+                                    </p>
+                                }
+                                colour={'orange'}
+                            ></FAQ>
+                            <FAQ
+                                question={'Does the club have a Facebook page?'}
+                                answer={
+                                    <p>
+                                        Yes! We have a{' '}
+                                        <a
+                                            href="https://www.facebook.com/compsci.adl/"
+                                            target="_blank"
+                                            rel="noopener noreferrer"
+                                            className="underline"
+                                        >
+                                            Facebook page
+                                        </a>{' '}
+                                        where we post all club events and general discussions.
+                                    </p>
+                                }
+                                colour={'yellow'}
+                            ></FAQ>
+                            <FAQ
+                                question={'How can I contact the CS Club?'}
+                                answer={
+                                    <p>
+                                        If you have any questions or just want to get in touch with
+                                        us, visit our{' '}
+                                        <a href="/contact" className="underline">
+                                            contact page
+                                        </a>{' '}
+                                        or send a message in our{' '}
+                                        <a
+                                            href="https://discord.gg/UjvVxHA"
+                                            target="_blank"
+                                            rel="noopener noreferrer"
+                                            className="underline"
```

Comment:
I think it's better to take the social links from the file containing our social links in the `data` directory. This way if we modify one of the links, we only have to update it in one place.